retrofit/src/main/java/retrofit2/http/Body.java

```diff
@@ -39,4 +39,9 @@
 @Target(PARAMETER)
 @Retention(RUNTIME)
 public @interface Body {
+  /**
+   * When true a {@code null} value will be ignored and an empty body will be used. Otherwise
```

Comment:
Ahh, so we have three cases:

 1. non-null
 2. null that maps to empty body
 3. null that is converted by the converter

Would it makes sense to punt case 2 to the converter?